Command.cpp

```diff
@@ -0,0 +1,220 @@
+#include "Command.h"
+#include <unordered_map>
+
+Model *Command::modelInstance = nullptr;
+
+void Command::setModelInstance(Model *instance) {
+  Command::modelInstance = instance;
+}
+
+const std::vector<std::string> Command::allCommands() {
+	std::vector<std::string> myvector{ "help", "help_for", "list", "save", "load", "exit", "",
```

Comment:
@Qpirons this should return a const reference and the vector should be marked static inside of this function

Reply:
thank you